src/main/java/pl/simpleascoding/tutoringplatform/domain/advertisement/Advertisement.java

```diff
@@ -33,15 +36,19 @@ public class Advertisement {
     @Size(min = 50, max = 255, message = "Description has to be between 50 and 255 characters")
     private String description;
 
+    @NotNull
+    private Instant creationDate;
```

Comment:
If you add `@CreationTimestamp` annotation to this field, Hibernate will automatically set the correct date while saving the entity.
You don't have to manually assign it.

Reply:
Right, thanks for advice